evm-contracts/test/v0.6/VRFD20.test.ts

```diff
@@ -0,0 +1,290 @@
+import { contract, setup, helpers, matchers } from '@chainlink/test-helpers'
+import { assert } from 'chai'
+import { ContractTransaction } from 'ethers'
+import { VRFD20Factory } from '../../ethers/v0.6/VRFD20Factory'
+import { VRFCoordinatorMockFactory } from '../../ethers/v0.6/VRFCoordinatorMockFactory'
+import { bigNumberify } from 'ethers/utils'
+
+let roles: setup.Roles
+let personas: setup.Personas
+const provider = setup.provider()
+const linkTokenFactory = new contract.LinkTokenFactory()
+const vrfCoordinatorMockFactory = new VRFCoordinatorMockFactory()
+const vrfD20Factory = new VRFD20Factory()
+
+beforeAll(async () => {
+  const users = await setup.users(provider)
+
+  roles = users.roles
+  personas = users.personas
+})
+
+describe('VRFD20', () => {
+  const deposit = helpers.toWei('1')
+  const fee = helpers.toWei('0.1')
+  const keyHash = helpers.toBytes32String('keyHash')
+  const seed = 12345
+
+  const requestId =
+    '0x66f86cab16b057baa86d6171b59e4c356197fcebc0e2cd2a744fc2d2f4dacbfe'
+
+  let link: contract.Instance<contract.LinkTokenFactory>
+  let vrfCoordinator: contract.Instance<VRFCoordinatorMockFactory>
+  let vrfD20: contract.Instance<VRFD20Factory>
+
+  const deployment = setup.snapshot(provider, async () => {
+    link = await linkTokenFactory.connect(roles.defaultAccount).deploy()
+    vrfCoordinator = await vrfCoordinatorMockFactory
+      .connect(roles.defaultAccount)
+      .deploy(link.address)
+    vrfD20 = await vrfD20Factory
+      .connect(roles.defaultAccount)
+      .deploy(vrfCoordinator.address, link.address, keyHash, fee)
+    await link.transfer(vrfD20.address, deposit)
+  })
+
+  beforeEach(async () => {
+    await deployment()
+  })
+
+  it('has a limited public interface', () => {
+    matchers.publicAbi(vrfD20Factory, [
+      // Owned
```

Comment:
Why are the commented-out attributes missing?

Reply:
That's a pattern in the tests, it's just to differentiate where in the inherritance tree these public methods come from.

Reply:
Yeah, they aren't attributes, @coventry. They're comments describing which contracts the subsequent attributes are inherited (and exposed) from.